.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -44,10 +44,19 @@ Closes <ticket>
 If there are any visual changes, screenshots should be added here. 
 -->
 
-## Required Approvals Before Merging
+## Testing and review
+
```

Comment:
@amyleadem I saw this on your PRs and thought it would be a good addition to the template. Is the description ok?